src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const filteredStylesList = sourceString
+    .split(';')
+    .filter((style) => /\S/.test(style));
+  const resultObject = {};
+
+  for (const styleItem of filteredStylesList) {
+    const keyValue = styleItem.split(':').map((item) => item.trim());
+    const [property, value] = [...keyValue];
+
+    resultObject[property] = value;
```

Comment:
The variable `resultObject` is named in a generic way which can decrease code readability. Consider using a more descriptive name that reflects its purpose, such as `stylesObject` or `cssProperties`, as per checklist item 2.